examples/timeline-charts-example/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
This should either be:
- Deleted (this folder isn't a git repo)
- Moved to the root of the git repo